apps/wallet-dashboard/components/Dialogs/settings/views/NetworkSelectorView.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import { Header, RadioButton } from '@iota/apps-ui-kit';
+import { DialogLayout, DialogLayoutBody } from '../../layout';
+import { NetworkConfiguration } from '@iota/iota-sdk/client';
+import { useNotifications } from '@/hooks';
+import { NotificationType } from '@/stores/notificationStore';
+import { useIotaClientContext } from '@iota/dapp-kit';
+
+interface NetworkSelectorViewProps {
+    handleClose: () => void;
+    onBack: () => void;
+}
+
+export function NetworkSelectorView({
+    handleClose,
+    onBack,
+}: NetworkSelectorViewProps): JSX.Element {
+    const { addNotification } = useNotifications();
+    const clientContext = useIotaClientContext();
+    const activeNetwork = clientContext.network;
+
+    async function handleNetworkChange(network: NetworkConfiguration) {
+        if (activeNetwork === network.id) {
+            return;
+        }
+        clientContext.selectNetwork(network.id);
+        addNotification(`Switched to ${network.name}`, NotificationType.Success);
```

Comment:
we should use `toast.success` here to have styled toasts :see_no_evil:  I'll open an issue to remove everything related to `addNotification`, `useNotifcations`,etc. but can you please update this to use toast? :pray:

Reply:
https://github.com/iotaledger/iota/issues/4515